harness/tests/localnet_test.go

```diff
@@ -76,14 +83,17 @@ func TestMain(m *testing.M) {
 }
 
 // TestSendBidTransaction ensures that a valid bid transaction has been accepted
-// by all nodes in the network within a particular time frame and within
-// the same block
+// by all nodes in the network within a particular time frame and within the
+// same block
 func TestSendBidTransaction(t *testing.T) {
 
 	walletsPass := os.Getenv("DUSK_WALLET_PASS")
 
-	t.Log("Send request to all nodes to loadWallet")
-	for i := 0; i < localNetSize; i++ {
+	// Half of the network is supposed to run the consensus, the rest should be
+	// light nodes
+	t.Log("Send request to half of the network nodes to loadWallet")
+	halfNetwork := localNetSize / 2
```

Comment:
nice, shall we do the same in the rusk-ipc branch ?

Reply:
To avoid merge conflicts maybe no.